estate/models/estate_property.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.tools import relativedelta
+
+def _compute_three_months_from_now():
+    return fields.Date.today() + relativedelta(months=3)
+
+class EstateProperty(models.Model):
+
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    active = fields.Boolean(default=True)
```

Comment:
This is already the default value of Boolean fields, it is redudant.

```suggestion
    active = fields.Boolean()
```